WISE-Community/WISE
# Moved dispatcherServletContexts.xml config to Java

How to test:
1. run mvn test and make sure that all tests pass
2. test WISE4 and WISE5 and make sure that they behave as before

Closes #1977
